src/ubuntu/18.04/crossdeps/Dockerfile

```diff
@@ -27,6 +27,9 @@ RUN apt-get update \
         libarchive-dev \
         libbsd-dev \
         libmpc-dev \
+        libxml2-utils \
+        cpio \
+        rpm2cpio \
```

Comment:
Please list the new packages in alphabetical order with the existing package listing.

Reply:
Thank you for the review I will fix it.